tests/conftest.py

```diff
@@ -0,0 +1,3 @@
+
+
+
```

Comment:
@avallbona I don't think we need to add this file, unless we were using it. Could you remove it?

Reply:
done